### ISSUE TITLE
oh-my-zsh plugin naming

### ISSUE THREAD

**Issue by jjangga0214** [zsh-users/zsh-history-substring-search#101]
README.md says to add oh-my-zsh plugin with name of **history-substring-search**. However, this is inconsistent with zsh-users' other plugins, such as **zsh-completions**, **zsh-syntax-highlighting** and **zsh-autosuggestions**, which are all prefixed by **zsh** and have same name with the repository. And actually, specifying its name as **zsh-history-substring-search** also works. So, why not change it?

**Comment by sunaku** [comment]
Yes, that is correct! :+1:  The plugin name originally included a "zsh-" prefix in #87, but it was later changed to remove the "zsh-" prefix in #91 because the prefix was causing installation problems for a user (@beheist).

If you can confirm that it's no longer a problem, please submit a PR that restores the "zsh-" prefix. Thanks!
